modules/virt-configuring-downward-metrics.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-exposing-downward-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-downward-metrics_virt-using-downward-metrics"]
+= Configuring downward metrics
+
+You enable the capturing of downward metrics for a host VM by creating a configuration file that includes the following options:
+
+* A `downwardMetrics` device
+* A `virtio-serial` port, through which the metrics are exposed
+
+You must also enable a `DownwardMetrics` feature gate.
+
+.Prerequisites
+* Administrator privileges are required to enable the feature gate.
+
+.Procedure
+
+. Edit or create a YAML file to expose the metrics by including a `downwardMetrics` device, a `virtio-serial` port, and the `DownwardMetrics` feature gate as shown in the following example:
++
+.Example downwardMetrics configuration file
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      downwardMetrics: {} <1>
+      disks:
+      - disk:
+          bus: virtio <2>
+        name: containerdisk
```

Comment:
This makes reference to the `containerdisk`, which is used for storing the VM boot image. This is not related with `downwardMetrics`.